CougSat1-ADCS/src/Configuration.h

```diff
@@ -2,8 +2,6 @@
 #define _SRC_CONFIGURATION_H_
 
 /*********************************** Timing ***********************************/
-#define PERIOD_MS_IDLE_SLEEP (10)    // Sleep time during idle
-#define PERIOD_MS_PERIODIC (5000)    // The periodic event
-#define PERIOD_MS_CONTROL_LOOP (200) // The control loop (update actuators)
-
+#define PERIOD_MS_POLLING_SLEEP (1)    // Minimum wait for polling
+#define MESSAGELENGTH (9) //9 byte messages will handle all possible communication
```

Comment:
Why have a fixed read length when each message is variable in length?

Reply:
We are setting the read length to the maximum length and expecting them to include whitespace.

Reply:
Including padding is a waste of resources. Read first byte, switch on it to figure out how many more to read.